apps/site/util/stringUtils.ts

```diff
@@ -28,3 +28,10 @@ export const dashToCamelCase = (str: string) =>
     // remove leftover - which don't match the above regex. Like 'es-2015'
     .replace(/-/g, '')
     .replace(/^[A-Z]/, chr => chr.toLowerCase());
+
+export const twoDateToUIID = (date1: string, date2: string): string => {
+  const date1String = date1.split('T')[0];
+  const date2String = date2.split('T')[0];
```

Comment:
nothing seems to enforce this as a a proper date string over within `site.json`, which could lead to bugs or errors here splitting on `T`

![image](https://github.com/user-attachments/assets/4e83288c-7c2f-4ea6-b819-a70db0ef6885)

Reply:
I'll have to reread the split docs because split automatically gives you a table that can be unique.

Reply:
But we can also use `new Date` and throw error if not right date